src/deadline/client/cli/_groups/bundle_group.py

```diff
@@ -321,8 +402,7 @@ def _check_create_job_wait_canceled() -> bool:
 )
 @click.option(
     "--submitter-name",
-    type=click.STRING,
-    help="Name of the application submitting the bundle. If a name is specified, the GUI will automatically close after submitting the job.",
+    help="[DEPRECATED] Use --submitter-info submitter_name=<name> instead. The name of the submitter.",
```

Comment:
Why not keep `--submitter-name` and have `--additional-submitter-info`? It's backwards compatible and submitter name is treated differently than the other submitter info.